shell/platform/darwin/macos/framework/Source/FlutterDartProject.mm

```diff
@@ -81,4 +81,38 @@ - (NSString*)ICUDataPath {
   return path;
 }
 
++ (NSString*)flutterAssetsName:(NSBundle*)bundle {
+  if (bundle == nil) {
+    bundle = [NSBundle bundleWithIdentifier:kAppBundleIdentifier];
+  }
+  if (bundle == nil) {
+    bundle = [NSBundle mainBundle];
+  }
+  NSString* flutterAssetsName = [bundle objectForInfoDictionaryKey:@"FLTAssetsPath"];
+  if (flutterAssetsName == nil) {
+    flutterAssetsName = @"Contents/Frameworks/App.framework/Resources/flutter_assets";
```

Comment:
I had a lot of trouble with this iOS hardcoded path when I was working on getting iOS app extensions working.  However you can make them match for now and we can address that later when we come to it.
https://github.com/flutter/engine/pull/40332/files#diff-aff27bb72cc390eb03f0b733e1fa490102e311ffac10e8e09e8245e582484220R79